### PR TITLE
feat(www): tweak twitterwall and link stuff

### PR DESCRIPTION
**Pull request by juliusmarminge**
Implements tweaks suggested in #2322

- [x] Clicking the tweet links to the tweet
- [x] Clicking the profile-part of the tweet links to the profile

Seems like there is some weird link behavior (children of link get underlined) set in some config so just gotta fix that

### CODE REVIEW

**Comment by ixahmedxi** [comment]
Tweets that have `&` render as `&amp;` on the website, can we somehow parse that text?

File: www/src/components/TwitterWall/tweets.ts [hunk 1]
```diff
@@ -1,98 +1,213 @@
...
+    },
+    {
+      id: '1548607105934004227',
+      name: 'Ben 🏳️‍🌈',
+      handle: 'nurodev',
+      date: 'Jul 17',
+      avatar:
+        'https://pbs.twimg.com/profile_images/1542996607083462659/sTlPB-kO_normal.jpg',
+      content:
+        '@alexdotjs @trpcio @github Amazing &amp; much well deserved!\nPeople are starting to see how awesome tRPC is &amp; I love it! 👏',
```
Comment (juliusmarminge) [inline www/src/components/TwitterWall/tweets.ts:62]: Interesting. That is actually coming from the API🤔

**Review by juliusmarminge — COMMENTED** [review]
Can look into this tonight

File: www/src/components/TwitterWall/script.ts [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    const userId = tweetData.user.id_str;
+
+    const content = tweetData.text;
+
+    // this removes the trailing line at the end of the tweet's content
+    const contentWithoutTrailingLink = content.replace(
+      /\b(http(s|):\/\/.+)\W*$/,
+      '',
+    );
+
```
Comment (juliusmarminge) [inline www/src/components/TwitterWall/script.ts:46]:
```suggestion
const escapedContent = content
    .replace(/&amp;/, '&')
    .replace(/&lt;/, '<')
    .replace(/&gt;/, '>')
    .replace(/&quot;/, '\"')
    .replace(/&#39;/, '\'');      

```

@ixahmedxi too hacky?

**Comment by juliusmarminge** [comment]
Not sure if we need to un-escape more than the ones I did now (like all the hundreds of special characters. if so we should probably find a better more scalable solution)? 

Only problem is if the tweet contains a link in the middle (which twitter shortens to their domain btw), that doesn't render as a link:

CleanShot 2022-07-28 at 15 57 00

**Review by KATT — APPROVED** [review]

**Comment by KATT** [comment]
Awesome! Thanks so much for this.

For the next iteration, I think a good idea would be to

- Feature specific tweets (probably the ones in the tweetsOld) and then be able to just use those, otherwise, we gotta keep this up-to-date and re-run the script all the time :)
- Unsure if we should have it dynamic at all based on the twitter love, but we should definitely link it in the bottom

**Pull request merged**